WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -768,6 +768,10 @@ private void SetFees()
 					{
 						FeePercentage = 100 * (decimal)EstimatedBtcFee.Satoshi / amount.Satoshi;
 					}
+					else
+					{
+						FeePercentage = 0;
```

Comment:
@yahiheb your change looks okay but I would like to ask you to try improve this ugly nest of if-else statements that do all exactly the same. Try replacing all with something like this:

```c#
long ParseAmount(string amountText) =>
	Money.TryParse(amountText.TrimStart('~', ' '), out Money value)
		? value.Satoshi
		: 0;

var amount = IsMax
	? selectedCoins.Sum(x => x.Amount)
	: ParseAmount(AmountText);

FeePercentage = amount != 0
	? 100 * (decimal)EstimatedBtcFee.Satoshi / amount
	: 0;
```

Reply:
Done but without the `ParseAmount` method.